### DISCUSSION TITLE
Idée amélioration aides : "tous pays"

### DISCUSSION THREAD

**Discussion by wittsarah** [betagouv/ellesfontlaphoto#267] [Sujets traités ou résolus]
Ajouter option "internationaux" dans la liste des conditions ? Plusieurs concours sont ouverts à tous (même les étrangers) mais la liste ne donne que des choix français ou option vide. Peut-être ajouter "tous pays" ou "européen" ?

**Comment by lauriane-f** [comment]
résolu - on peut laisser le champ "residence-condition" et les aides concernées apparaîtront quelque soit le filtre sélectionné
